plugins/azure_blob_storage/requirements.txt

```diff
@@ -1,6 +1,6 @@
 # List third-party dependencies here, separated by newlines. 
 # All dependencies must be version-pinned, eg. requests==1.2.0
 # See: https://pip.pypa.io/en/stable/user_guide/#requirements-files
-requests==2.22.0
-xmltodict==0.12.0
-parameterized==0.8.1
+xmltodict==0.13.0
+parameterized==0.9.0
+requests==2.32.2
```

Comment:
I think we can remove requests as it is included in our insightconnect-plugin-runtime.

Reply:
👍 done, thanks!